code/game/objects/items/stacks/medical.dm

```diff
@@ -15,54 +15,43 @@
 	///Fumble delay applied without sufficient skill
 	var/unskilled_delay = SKILL_TASK_TRIVIAL
 
-/obj/item/stack/medical/attack(mob/living/carbon/M as mob, mob/user as mob)
-	if(!istype(M))
-		to_chat(user, span_warning("\The [src] cannot be applied to [M]!"))
-		return TRUE
+/obj/item/stack/medical/attack(mob/living/M, mob/living/user)
+	. = ..()
+	if(.)
+		return
 
-	if(M.status_flags & INCORPOREAL || user.status_flags & INCORPOREAL) //Incorporeal beings cannot attack or be attacked
-		return TRUE
+	if(!ishuman(M))
+		M.balloon_alert(user, "not a human")
+		return FALSE
+	var/mob/living/carbon/human/target = M
 
-	if(!ishuman(user))
-		to_chat(user, span_warning("You don't have the dexterity to do this!"))
-		return TRUE
+	if(!ishuman(user)) // TODO port advanced_tool_user
```

Comment:
Not that it actually matters, but shouldn't we look for the dexterous flag instead?